projects/plugins/jetpack/_inc/client/traffic/style.scss

```diff
@@ -137,6 +137,8 @@
 }
 
 .jp-seo-social-previews {
+	margin-bottom: 0 !important;
```

Comment:
It seems like `.jp-seo-social-previews` already has enough specificity to not need `!important`.

Reply:
Oh, I was following the way it's done in other blocks, I'll remove redundant `!important` now, thanks!